Cauldron/Assets/Scripts/UI/Menu/AuthorsMenu.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Cysharp.Threading.Tasks.Triggers;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace CauldronCodebase
+{
+    public class AuthorsMenu : MonoBehaviour
+    {
+        [SerializeField] private Button closePanelButton;
+        [Header("Fade In Out")]
+        [SerializeField] private FadeController fade;
+
+        private void OnValidate()
+        {
+            if (!fade) fade = FindObjectOfType<FadeController>(true);
```

Comment:
It's a nice way, but we have Zenject in our project, find DontDestroyInstaller and try to use it.